packages/@react-spectrum/tabs/test/Tabs.test.js

```diff
@@ -134,6 +140,33 @@ describe('Tabs', function () {
     expect(arrowDown.defaultPrevented).toBe(false);
   });
 
+  it('allows user to change tab item select via arrow keys with horizontal tabs (rtl)', async function () {
+    let onKeyDown = jest.fn();
+    let container = renderComponent({orientation: 'horizontal', providerProps: {locale: 'ar-AE'}});
+    let tabsTester = testUtilUser.createTester('Tabs', {root: container.getByRole('tablist'), interactionType: 'keyboard', direction: 'rtl'});
+    let tabs = tabsTester.tabs;
+    window.addEventListener('keydown', onKeyDown);
+
+    expect(tabs[0]).toHaveAttribute('aria-selected', 'true');
+
+    await tabsTester.triggerTab({tab: 1});
+    expect(tabs[0]).not.toHaveAttribute('aria-selected', 'true');
+    expect(tabs[1]).toHaveAttribute('aria-selected', 'true');
+    // Just to double check that the util is actually pressing the expected arrow key
+    expect(onKeyDown.mock.calls[0][0].key).toBe('ArrowLeft');
```

Comment:
I appreciate checking this, however, I'm a little reluctant to have it here as it's testing our test utils, not the component.
I'd kinda prefer it to be in its own test suite for the test utilities, though I understand this is faster to get checked out. it'll just tie our implementations all together making later updates harder potentially

Reply:
yep, makes sense, I'll make a note to make some test util specific tests